gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/DocumentReferenceMapper.java

```diff
@@ -0,0 +1,218 @@
+package uk.nhs.adaptors.pss.translator.mapper;
+
+import static uk.nhs.adaptors.pss.translator.util.ResourceUtil.buildIdentifier;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.dstu3.model.Attachment;
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.DocumentReference;
+import org.hl7.fhir.dstu3.model.Encounter;
+import org.hl7.fhir.dstu3.model.Enumerations;
+import org.hl7.fhir.dstu3.model.InstantType;
+import org.hl7.fhir.dstu3.model.Organization;
+import org.hl7.fhir.dstu3.model.Patient;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.v3.RCMRMT030101UK04Component4;
+import org.hl7.v3.RCMRMT030101UK04EhrComposition;
+import org.hl7.v3.RCMRMT030101UK04EhrExtract;
+import org.hl7.v3.RCMRMT030101UK04NarrativeStatement;
+import org.springframework.stereotype.Service;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import uk.nhs.adaptors.pss.translator.util.DateFormatUtil;
+import uk.nhs.adaptors.pss.translator.util.EhrResourceExtractorUtil;
+import uk.nhs.adaptors.pss.translator.util.EncounterReferenceUtil;
+import uk.nhs.adaptors.pss.translator.util.ParticipantReferenceUtil;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class DocumentReferenceMapper {
+
+    //TODO Add custodian using the reference to Organisation that is generated for sending Practice
+    //TODO Add file Size using the uncompressed/unencoded size of the document
```

Comment:
Are these still to be implemented in this ticket or something for a later ticket? If so could you create tickets for these and add the ticket numbers to these comments?